app.js

```diff
@@ -901,7 +901,7 @@ io.sockets.on('connection',function(socket){
                 openhab.status = 'online';
                 openhab.last_online = new Date;
                 openhab.openhabVersion = socket.handshake.openhabVersion;
-                openhab.myohVersion = socket.handshake.myohVersion;
+                openhab.clientVersion = socket.handshake.clientVersion;
```

Comment:
we could also let the myohVersion remain and add the clientVersion on top? Another way is to clean it out.